test/PlaidClientTest.js

```diff
@@ -591,8 +440,9 @@ describe('plaid.Client', () => {
           });
         });
 
-        it('all transactions (promise)', cb => {
-          pCl.getAllTransactions(accessToken, now, now).then(transactions => {
+        it.skip('all transactions (promise)', cb => {
```

Comment:
Can we document why these are skipped?

Reply:
:zap:

Opened https://github.com/plaid/plaid-node/issues/186 and referenced that in a comment